lacommunaute/forum_moderation/tests/test_topic_delete_view.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+from django.urls import reverse
+
+from lacommunaute.forum.factories import ForumFactory
+from lacommunaute.forum_conversation.factories import TopicFactory
+
+
+@pytest.fixture
+def topic():
+    return TopicFactory(forum=ForumFactory(with_public_perms=True), with_post=True)
+
+
+def test_get_on_topic_delete_view(client, db, topic, monkeypatch):
+    def mock_return(*args, **kwargs):
+        return True
+
+    url = reverse("forum_moderation_extension:topic_delete", kwargs={"slug": topic.slug, "pk": topic.pk})
+    client.force_login(topic.poster)
+    monkeypatch.setattr("lacommunaute.forum_moderation.views.TopicDeleteView.perform_permissions_check", mock_return)
+
+    response = client.get(url)
+    assert response.status_code == 200
+    assert (
+        '<div class="mb-3 warning-message">Voulez-vous vraiment supprimer ce sujet ?</div>'
+        in response.rendered_content
+    )
```

Comment:
[_pytest-django_ expose les assertions de Django](https://pytest-django.readthedocs.io/en/latest/helpers.html#assertions)
```suggestion
    assertContains(response, '<div class="mb-3 warning-message">Voulez-vous vraiment supprimer ce sujet ?</div>', html=True)
```

Reply:
Quel est le bénéfice à utiliser les assertions Django dans ce cas ?

Reply:
Dans le cas présent c'est assez faible, même si ça évite _un peu_ de "réinventer la roue". Ça permet surtout de rafraîchir leur existence pour les prochains devs (par exemple quand il faut tester les messages :D), que les copiés/collés ne dégénère pas trop, de ne pas multiplier les manières de faire..